### PR TITLE
Add doku for networking of virtualization

### PR DESCRIPTION
**Pull request by hardikdr**
@byteocean  Do you want to resolve the conflicts from your branch and so that we can add your changes to the main branch?
It'll help as we will then not have "todo" marks on the main page.

### CODE REVIEW

**Comment by hardikdr** [comment]
Thanks, @byteocean .
Looks like there are some linting errors, that need to be resolved.
I think golint and the extension in vscode should help in finding and resolving them.

File: docs/components/virtualization/HowToDeploy.md [hunk 1]
```diff
@@ -11,25 +11,14 @@ This document walks through how to deploy the virtualization stack on a baremeta
 ## At a glance
```
Comment (afritzler) [inline docs/components/virtualization/HowToDeploy.md:11]: Can we please use lower case file names? Also I would name it like `installation.md` or `usage.md`.

File: docs/components/virtualization/Overview.md [hunk 2]
```diff
@@ -1,2 +1 @@
 # Overview
-
```
Comment (afritzler) [inline docs/components/virtualization/Overview.md:2]: Same here please use lower case file names.

**Review by afritzler — CHANGES_REQUESTED** [review]
Please change the file names and fix the reference in the main `mkdocs.yml` file in the project root.

File: docs/components/virtualization/HowToDeploy.md (hunk 1 above)
Comment (hardikdr) [reply docs/components/virtualization/HowToDeploy.md:11]: Sure, that's a good point, done.

File: docs/components/virtualization/Overview.md (hunk 2 above)
Comment (hardikdr) [reply docs/components/virtualization/Overview.md:2]: done

**Review by afritzler — APPROVED** [review]

**Comment by afritzler** [comment]
Looks like this can be merged now.

**Pull request merged**